spec/models/availability_spec.rb

```diff
@@ -38,12 +46,16 @@
     context 'An event starts before the range and ends in the range' do
       it 'is not available' do
         user = create :user
-        availability = Availability.create(week_day: DateTime.new(2015, 03, 06, 20, 00).wday, start_time: DateTime.new(2015, 03, 06, 20, 00), end_time: DateTime.new(2015, 03, 06, 23, 30), user_id: user.id)
-        event = Event.new(name: 'random event', location: 'Braga', start_date: DateTime.new(2015, 03, 06, 19, 00), end_date: DateTime.new(2015, 03, 06, 21, 00), owner: user.id)
+        date_time_now = DateTime.now
+        one_hour_earlier = date_time_now + 1.hour
+        two_hours_later = date_time_now + 2.hour
+        three_hours_later = date_time_now + 3.hour
+        availability = Availability.create(week_day: date_time_now.wday, start_time: date_time_now, end_time: three_hours_later, user_id: user.id)
+        event = Event.new(name: 'random event', location: 'Braga', start_date: one_hour_earlier, end_date: two_hours_later, owner_id: user.id)
 
         result = availability.include?(event.start_date)
 
-        expect(result).to be(false)
+        expect(result).to be(true)
```

Comment:
:+1:  I was setting the `one_hour_earlier = date_time_now + 1.hour` instead of `one_hour_earlier = date_time_now - 1.hour` and for some reason i thought that it should be true.